### ISSUE TITLE
Add a Light colored Theme

### ISSUE THREAD

**Issue by GrazianoCapelli** [BasicAirData/Clinometer#21]
A dark theme is a Nightmare to use in bright Environments.
I propose to implement a light Theme to improve the readability in these conditions.
The user should have the possibility to switch between light and dark Theme during the usage.

We should start here with a UI mockup.

**Comment by Jenish-1235** [comment]
I can add light theme as well as any other color theme support in app. You can assign this issue to me.

**Comment by GrazianoCapelli** [comment]
I updated the header comment in order to better describe what we would like to implement.
Please read it carefully and, if you think that it is good for you, give me a feedback.
I would be be glad to assign this issue to you.

If you work on more than 1 issue, please use a different branch for each one.
You can read the CONTRIBUTING.md file.

PS: Alternative UI mockups are harmly welcome.

**Comment by Jenish-1235** [comment]
Yes, definitely. I will be using separate branches for separate issues. I can accomplish this task. You can assign this to me.

**Comment by Jenish-1235** [comment]
I added light colored theme and preference button in settings menu , do have a look at that and link my pr to issue.
